app-crypt/efitools/efitools-1.8.1.ebuild

```diff
@@ -12,7 +12,7 @@ SRC_URI="https://git.kernel.org/pub/scm/linux/kernel/git/jejb/efitools.git/snaps
 LICENSE="GPL-2 LGPL-2.1"
 SLOT="0"
 KEYWORDS="~amd64 ~arm64 ~x86"
-IUSE="libressl"
+IUSE="libressl static"
 
 RDEPEND="!libressl? ( dev-libs/openssl:0= )
```

Comment:
you'll need to update these library deps.  look at wget as an example of how to deal with static/static-libs sanely (the LIB_DEPEND logic).

Reply:
Done in the updated branch.